src/Swarm/Language/Capability.hs

```diff
@@ -37,6 +37,8 @@ data Capability
     CPower
   | -- | Execute the 'Move' command
     CMove
+  | -- | Execute the 'Move' command for a heavy robot
+    CMoveheavy
```

Comment:
Looking into the thesaurus, "locomote" or "mobile" sound good to me, but "move-heavy" is at least literal. :slightly_smiling_face:

Reply:
Yeah, since users will never see these names at all, I'd prefer to make it more literal.